documentation/modules/ROOT/pages/operations/debezium-server.adoc

```diff
@@ -478,6 +478,48 @@ The configuration will need to contain the list of predicates, implementation cl
 
 |===
 
+[id="debezium-async-engine-properties"]
+=== Asynchronous Engine Properties
+
+By default, Debezium server uses `AsyncEmbeddedEngine` as the processing engine.
+You can configure following options for asynchronous embedded engine:
+
+[cols="35%a,10%a,55%a"]
+|===
+|Property
+|Default
+|Description
+
+|`record.processing.threads`
+|The number of available cores.
```

Comment:
```suggestion
|`AVAILABLE_CORES` (the number of cores available on the CPU)
```

Reply:
Actually, this doesn't seem right. If the default is to use the number of available cores, then it would make sense that this would be the same thing as specifying the placeholder value. However, later in the description it implies a different default: i.e., "If the value is not specified, threads are created as needed ..."

Reply:
When this option is omitted, Debezium uses a caching thread pool strategy, where threads are created based on demand and reused. If a thread remains unused in the cache for more than 60 seconds, it is terminated and removed.

When the option is provided, we use a fixed thread pool based on the interpreted value. The interpreted value is then either the number of cores you've specified in the option or the placeholder, where we will resolve the number of cores to be used based on the number of cores available to the JVM runtime, which allows Debezium to react across restarts in an elastic way if you increase or reduce the number of cores to the environment.

Reply:
Ok, do I understand that you must literally specify the placeholder value? I had been thinking that when we talked about the placeholder that it was functionally equivalent to the default value.